presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -248,22 +248,33 @@ private static KeyProvider toKeyProvider(DwrfProto.Encryption.KeyProvider keyPro
         }
     }
 
-    private static List<StripeInformation> toStripeInformation(List<DwrfProto.StripeInformation> types)
-    {
-        return ImmutableList.copyOf(Iterables.transform(types, DwrfMetadataReader::toStripeInformation));
+    private static List<StripeInformation> toStripeInformation(List<DwrfProto.StripeInformation> stripeInformations)
+    {
+        ImmutableList.Builder<StripeInformation> stripeInfoBuilder = ImmutableList.builder();
+        List<byte[]> previousKeysMetadata = ImmutableList.of();
+        for (DwrfProto.StripeInformation dwrfStripeInfo : stripeInformations) {
+            StripeInformation prestoStripeInfo = toStripeInformation(dwrfStripeInfo, previousKeysMetadata);
```

Comment:
nits: maybe ```currentStripeInfo``` instead of ```prestoStripeInfo``` and ```previousKeyMetadata``` instead of ```previousKeysMetadata```

Reply:
for currentStripeInfo vs. prestoStripeInfo, I was trying to capture that we were converting from DwrfProto.StripeInformation object to Presto's stripeInformation logic, so I'll leave that name as is.  Changed previousKeyMetadata though.